curvefs/src/client/fuse_volume_client.cpp

```diff
@@ -298,13 +274,23 @@ CURVEFS_ERROR FuseVolumeClient::Truncate(Inode *inode, uint64_t length) {
 
 CURVEFS_ERROR FuseVolumeClient::FuseOpFlush(fuse_req_t req, fuse_ino_t ino,
                                             struct fuse_file_info *fi) {
-    // Todo: flush data and note cto
-    return CURVEFS_ERROR::OK;
+    VLOG(9) << "FuseOpFlush, ino: " << ino;
```

Comment:
curvefs::client::common::FLAGS_enableCto = true,  FuseOpFlush need to sync inode for cto(close will call flush internal)